dallinger/recruiters.py

```diff
@@ -1,9 +1,8 @@
 """Recruiters manage the flow of participants to the experiment."""
-
 import os
 
 from boto.mturk.connection import MTurkConnection
-from psiturk.models import Participant
+from dallinger.models import Participant
```

Comment:
It appears changing this back breaks some tests. Both of these:

    tests.test_demos.TestBartlett1932.test_instantiation
    tests.test_demos.TestBartlett1932.test_networks_holds_single_experiment_node

fail with:

    ERROR - no config.txt file in the current directory. 

    Are you use this directory is a valid psiTurk experiment?  If you are starting a new project run 'psiturk-setup-example' first.

When the import is from `psiturk.models`, it seems like importing from psiturk.models has some side effects that set the expectation that the code is run from an experiment directory.